src/vs/base/browser/ui/list/listView.ts

```diff
@@ -441,7 +442,12 @@ export class ListView<T> implements IListView<T> {
 
 		// Prevent the monaco-scrollable-element from scrolling
```

Comment:
Comment is wrong. Would be helpful to have some comments explaining how this works and why it's a bit odd